tests/rule_based_profiler/parameter_builder/test_mean_unexpected_metric_multi_batch_parameter_builder.py

```diff
@@ -0,0 +1,279 @@
+from typing import Any, Dict, Optional
+
+import numpy as np
+import pytest
+
+from great_expectations import DataContext
+from great_expectations.execution_engine.execution_engine import MetricDomainTypes
+from great_expectations.rule_based_profiler.helpers.util import (
+    get_parameter_value_and_validate_return_type,
+)
+from great_expectations.rule_based_profiler.parameter_builder import (
+    MetricMultiBatchParameterBuilder,
+)
+from great_expectations.rule_based_profiler.parameter_builder.mean_unexpected_metric_multi_batch_parameter_builder import (
+    MeanUnexpectedMapMetricMultiBatchParameterBuilder,
+)
+from great_expectations.rule_based_profiler.types import Domain, ParameterContainer
+from tests.rule_based_profiler.conftest import ATOL, RTOL
+
+
+def test_instantiation_mean_unexpected_metric_multi_batch_parameter_builder(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    data_context: DataContext = (
+        bobby_columnar_table_multi_batch_deterministic_data_context
+    )
+
+    _: MeanUnexpectedMapMetricMultiBatchParameterBuilder = (
+        MeanUnexpectedMapMetricMultiBatchParameterBuilder(
+            name="my_name",
+            map_metric_name="column_values.nonnull",
+            total_count_parameter_builder_name="my_total_count",
+            data_context=data_context,
+        )
+    )
```

Comment:
Nitpick - we don't even need assignment here right? Perfectly fine to keep this but we should probably have a consistent approach since I tend to omit assignment when I'm just testing if the behavior errors/logs/etc.